config/locales/forms.yml

```diff
@@ -170,9 +180,16 @@ en:
           "yes": "Yes"
 
       jobseekers_job_application_personal_details_form:
+        your_address_html: Your address (<span class='text-red'>Required</span>)
+        building_and_street: Building and street
+        email_address_html: Email address (<span class='text-red'>Required</span>)
         first_name_html: First name (<span class='text-red'>Required</span>)
         last_name_html: Last name (<span class='text-red'>Required</span>)
-
+        national_insurance_number_html : National insurance number (<span class='text-red'>Required</span>)
```

Comment:
`insurance` or `Insurance`?

Reply:
It's lower case in the design!

Reply:
Content in the design is not 100% to be trusted